File: dependencies/infrastructure/src/main/java/com/foreveross/atwork/infrastructure/model/organizationSetting/WatermarkSettings.java
package com.foreveross.atwork.infrastructure.model.organizationSetting;

import android.os.Parcel;
import android.os.Parcelable;

import com.google.gson.annotations.SerializedName;

/**
 * Created by dasunsy on 2017/11/16.
 */

public class WatermarkSettings implements Parcelable {

    @SerializedName("organization")
    public String mOrganization;

    @SerializedName("employee")
    public String mEmployee;


    public WatermarkSettings() {
    }

    @Override
    public int describeContents() {
        return 0;
    }

    @Override
    public void writeToParcel(Parcel dest, int flags) {
        dest.writeString(this.mOrganization);
        dest.writeString(this.mEmployee);
    }

    protected WatermarkSettings(Parcel in) {
        this.mOrganization = in.readString();
        this.mEmployee = in.readString();
    }

    public static final Creator<WatermarkSettings> CREATOR = new Creator<WatermarkSettings>() {
        @Override
        public WatermarkSettings createFromParcel(Parcel source) {
            return new WatermarkSettings(source);
        }

        @Override
        public WatermarkSettings[] newArray(int size) {
            return new WatermarkSettings[size];
        }
    };
}

File: dependencies/infrastructure/src/main/java/com/foreveross/atwork/infrastructure/model/domain/WatermarkSettings.kt
package com.foreveross.atwork.infrastructure.model.domain

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class WatermarkSettings(
        @SerializedName("user")
        var user: String = "none",

        @SerializedName("discussion")
        var discussion: String = "none",

        @SerializedName("bing")
        var bing: String = "none",

        @SerializedName("email")
        var email: String = "none",

        @SerializedName("favorites")
        var favorites: String = "none"


) : Parcelable